web/yo/app/views/drugs.html

```diff
@@ -0,0 +1,39 @@
+<div class="drugs">
+        <h3 class="headerStyle">Curated Drugs</h3>
+        <table class="table" datatable="ng" dt-options="dtOptions" dt-columns="dtColumns">
+            <thead>
+            <tr class="dashedHeader">
+                <th>Name</th>
+                <th>NCI Thesaurus Code</th>
+                <th>Description</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr data-ng-repeat="drug in drugList">
+                <td ng-include="getTemplate(drug)">
+                </td>
+                <td>{{ drug.NCI }}</td>
+                <td>{{ drug.description }}</td>
+            </tr>
+            </tbody>
+        </table>
+
+        <div class="col-sm-9">
+            <input type="text" class="form-control" ng-model="suggestedDrug" uib-typeahead="drug as drug for drug in processSearchDrugs($viewValue)" typeahead-min-length="1"/>
```

Comment:
1) processSearchDrugs returns a list of string, so you could simply just use drug in processSearchDrugs($viewValue)
2) Let's also display synonyms here under the drug name. Similarly to what we have on the public website, we can discuss in details.